firebase.json

```diff
@@ -25,7 +25,25 @@
           "rewrites": [ {
             "source": "**",
             "destination": "/index.html"
-          }]
+            } ],
+            "headers": [ {
```

Comment:
Add the same entry in the other two objects as well (prod and uat).
Also please update the formatting and indentation changes to follow this files pattern.

Reply:
Sure I will change.